README.md

```diff
@@ -20,39 +23,28 @@ You have to set the environment variables **`TB_KEY`** and **`TB_SECRET`**. Alte
 ## Test running
 
 
-WDIO has different commands to run tests, you can tun a single test, a set of tests or all tests. By default tests are run in a headless manner meaning no browser window will be opened while running the tests and results will be seen in the terminal.
-
-  
+WebdriverIO has different commands to run tests, you can run a single test, a set of tests or all tests. By default tests are run in a headless manner meaning no browser window will be opened while execution and results will be seen in the terminal.
 
 You will find more detailed information [here](https://webdriver.io/docs/organizingsuites/).
 
-  
-
 To get started, with the following command you will be able to run all your tests.
 
-  
-
 	npm run wdio
 
-  
-
-**Note**: This command is specified in *package.json* file, to run tests with different options, you will have to look at the documentation and modify the command you are using.
+**Note**: This command is specified in *package.json* file, to run tests with different options, you will have to read the documentation and modify the command you are using.
 
 ## Folder organization
 
-If you have a look at the folder organization, you will find the following:
+If you have a look at the folder organization, you will find the following structure:
 
 -  **TEST:**
 
-Inside this folder, you will have components, features and step-definitions.
+Inside this folder, you will find components, features and step-definitions.
 
 -  **COMPONENTS**:
-
   
 Each part of the web is divided in components, so you can detail each one without mixing them. Inside here usually you will find a folder for each part (examples are shown in the project).
 
-  
-
 Inside each component, you will have the file for the functions, and also a file where you will write all the selectors/or locators you need to find the elements.
 
 **Note**: When code is repeating or you think that you will use it in another function, a component usually named shared is created. There, you will follow the same file organization, but for things that you will share between other components (data, functions, etc).
```

Comment:
I would rewrite this to make it clear that having anything inside "shared" is heavily discouraged and should only be used when unavoidable.

Reply:
True, I am working on that